README.md

```diff
@@ -70,6 +70,38 @@ Although Spring Zeebe has a transitive dependency to the [Zeebe Java Client](htt
 </dependency>
 ```
 
+The following compiler flag `-parameters` is required for Spring-Zeebe versions higher than 8.3.1:
```

Comment:
but only if you want to use @VAriables - right? Then I would be specific here - then users can decide to not add that flag and just not use Variables

Reply:
Yup! Added explicit note [here](https://github.com/camunda-community-hub/spring-zeebe/pull/514/commits/751b318461526dd060a769764a32931f833c2732)